awx/ui_next/src/screens/Login/Login.jsx

```diff
@@ -147,7 +147,7 @@ function AWXLogin({ alt, isAuthenticated }) {
     <Login header={Header} footer={Footer}>
       <LoginMainHeader
         data-cy="login-header"
-        title={brandName ? t`Welcome to Ansible ${brandName}!` : ''}
```

Comment:
@jakemcdermott, I also looked for other parts of the code that we are using `brandName` and the only other one similar case that I found was on the `document.title` - updated accordingly.

Reply:
sgtm